### ISSUE TITLE
Extra arguments for IfError

### ISSUE THREAD

**Issue by repton-infinity** [MicrosoftDocs/powerapps-docs#1496]
Can you give an example of how the extra arguments to IfError are useful?
e.g. this: **IfError(1/0,1,2/0,2)** returns **1**. As far as I can tell, it will have evaluated 2/0 and replaced it with 2, but the function only returns a single value, so the 2 is lost. What is something useful I could do with it that involves four or more arguments?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7e0983e-5863-d1eb-43e5-e6e9546c4e2b
* Version Independent ID: cc7abb21-14fd-04de-e1bd-2605697adcd3
* Content: [IfError and IsError functions in Power Apps - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/functions/function-iferror)
* Content Source: [powerapps-docs/maker/canvas-apps/functions/function-iferror.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/maker/canvas-apps/functions/function-iferror.md)
* Service: **powerapps**
* GitHub Login: @gregli-msft
* Microsoft Alias: **gregli**

**Comment by gregli-msft** [comment]
The extra arguments are useful to stop further processing.  See the section **Stopping further processing** in the article for details.  In your example, 2/0 will not be evaluated and that's important if this formula had side effects.

**Comment by HairyDrumroll** [comment]
> Can you give an example of how the extra arguments to IfError are useful? e.g. this: **IfError(1/0,1,2/0,2)** returns **1**. As far as I can tell, it will have evaluated 2/0 and replaced it with 2, but the function only returns a single value, so the 2 is lost. What is something useful I could do with it that involves four or more arguments?

I found the documentation pretty hard to follow, but this is how I've used it:
`IfError(
    callFirstFunction(); false, 
    Notify("First Function failed: " & FirstError.Kind & " - " & FirstError.Message, NotificationType.Error); true,
    callNextDependantFunction(); false,
    Notify("Dependant Function failed: " & FirstError.Kind & " - " & FirstError.Message, NotificationType.Error)); true,
    false)`
